mozilla/foundation.mozilla.org
# added soft-char limit widget to product category name

Closes #7789

**Link to sample test page**: https://foundation-s-7789-exten-vidgun.herokuapp.com/en/cms/snippets/wagtailpages/buyersguideproductcategory/edit/3/

**Steps to test**:

1. Visit the edit page linked above ([review app details](https://mozilla.slack.com/archives/CBKP2SG94/p1639007605007600)):
2. Note the addition of the character limit widget on the bottom right of the "name" field. It should work as expected however since this is a soft limit, it will still let you save if you go over the limit.
3. If everything is working as expected, testing is complete!
